### PR TITLE
refactor: Specify user explicitly for all services

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?
Not all services have the user specified explicitly.
The CIS/DBS tool expects that when running as root that be explicit.

## What is the new behavior?
All services, except Kong, have the user specified explicitly.
Kong runs as `kong` user, but doesn't expose it so it can be used in the compose file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/add-security.yml [hunk 1]
```diff
@@ -171,6 +175,7 @@ services:
 
   kong:
     image: kong:${KONG_VERSION}
+    #user: "kong:kong" # Note Kong already running as kong user but doesn't expose kong user, so this fails
```
Comment (jim-wang-intel) [inline compose-builder/add-security.yml:178]: from the Dockerfile of kong it actually runs as `kong:nogroup`  and they never create a group to run with: https://github.com/Kong/docker-kong/blob/master/alpine/Dockerfile#L40-L44   
They only created user but not group

Comment (lenny-intel) [reply compose-builder/add-security.yml:178]: Ahh, great info.  I'll try "kong:nogroup". Thanks!

Comment (jim-wang-intel) [reply compose-builder/add-security.yml:178]: maybe try `kong:0`?

Comment (lenny-intel) [reply compose-builder/add-security.yml:178]: `nogroup` works and it is more explicit.

Comment (jim-wang-intel) [reply compose-builder/add-security.yml]: awesome

**Review by jim-wang-intel — APPROVED** [review]

**Pull request merged**